src/yunohost/domain.py

```diff
@@ -397,13 +408,23 @@ def _build_dns_conf(domain, ttl=3600):
         ["@", ttl, "CAA", '128 issue "letsencrypt.org"']
     ]
 
-    return {
+    # Official record
+    res = {
         "basic": [{"name": name, "ttl": ttl, "type": type_, "value": value} for name, ttl, type_, value in basic],
         "xmpp": [{"name": name, "ttl": ttl, "type": type_, "value": value} for name, ttl, type_, value in xmpp],
         "mail": [{"name": name, "ttl": ttl, "type": type_, "value": value} for name, ttl, type_, value in mail],
         "extra": [{"name": name, "ttl": ttl, "type": type_, "value": value} for name, ttl, type_, value in extra],
     }
 
+    # Custom record
+    hookres = hook_callback('custom_dns_rules', args=[domain])
+    for n, val in hookres.items() :
```

Comment:
Can you please use full name variable? This makes the code harder to read, especially on the for/list comprehension line (and code is more read and wrote etc...)